novem/cli/__init__.py

```diff
@@ -339,9 +339,9 @@ def run_cli_wrapped() -> None:
         print(f"Proxy settings: {proxy}")
         hname = socket.gethostname()
         print(f"Hostname: {hname}")
-        api_ip = socket.gethostbyname("api.novem.no")
+        api_ip = socket.gethostbyname("api.novem.io")
         print(f"API IP : {api_ip}")
-        main_ip = socket.gethostbyname("novem.no")
+        main_ip = socket.gethostbyname("novem.io")
```

Comment:
It would be super nice if it also read the current default API_ROOT from the config and printed both the url and the resulting IP address

Reply:
fix l8r